src/Integration.Vsix/SonarLintIntegrationPackage.cs

```diff
@@ -80,6 +81,8 @@ protected override async System.Threading.Tasks.Task InitializeAsync(Cancellatio
             await base.InitializeAsync(cancellationToken, progress);
             await JoinableTaskFactory.SwitchToMainThreadAsync();
             await InitOnUIThreadAsync();
+            
```

Comment:
Will wait to remove this until the review is done, just so i dont have a single commit to remove white space if theres more stuff to do

Reply:
To be honest, I wouldn't bother - we're going to make changes to this file again, and we could do it then.